packages/ethash/README.md

```diff
@@ -0,0 +1,110 @@
+# @ethereumjs/ethash
+
+[![Coverage Status](https://img.shields.io/coveralls/ethereumjs/ethashjs.svg)](https://coveralls.io/r/ethereumjs/ethashjs)
+
+[![NPM Package][vm-npm-badge]][vm-npm-link]
```

Comment:
nice! looks like needs to be updated to ethash-*

Reply:
absolutely, that's pretty much a work in progress. Will resume it on monday.